roadmap.html.haml

```diff
@@ -12,41 +12,47 @@ title: Road Map
 
 .row
   .col-md-12
-    %h2 Infinispan 9.x (under development)
-    %p
+    %p.text-center
       %img{:src => "/images/roadmap.svg"}
     
 .row
+  .col-md-12
+    %h3.alert.alert-warning.text-center Warning: The list below is subject to change without notice at any time.
   .col-md-6
     %i.enormous-icon.fa-wrench.pull-left.icon-2x
     %h3 API
     %ul
-      %li Publish Subscribe / Topics [9.2]
+      %li New Modular API
+      %li Publish Subscribe / Topics
     %i.enormous-icon.fa-bullseye.pull-left.icon-2x
     %h3 Core
     %ul
-      %li Performance improvements [9.x]
-      %li Data transcoding [9.2]
-      %li Pre-node shutdown checks to ensure that the remaining cluster will have enough resources to continue [9.x]
-      %li Compatibility with both Java 8 and Java 9 [9.2]
+      %li In-place rolling upgrades
+      %li Continued performance and resource improvements
     %i.enormous-icon.icon-question-sign.pull-left.icon-2x
     %h3 Query
     %ul
-      %li Continuous query materialized view, aggregations, indexed support for initial state [9.x]
       %li ProtoBuf 3 support for remote queries
+    %i.enormous-icon.fa-database.pull-left.icon-2x
+    %h3 Persistence
+    %ul
+      %li Striped cache stores for faster iteration
+      %li Async persistence layer
```

Comment:
I am not 100% that we will get this in this version. We will have async iteration at least :D